### PR TITLE
Ignore SigPipe by default

### PR DESCRIPTION
**Pull request by domenukk**
If you talk to a pipe that quits, it shouldn't kill the fuzzer

### CODE REVIEW

File: libafl/Cargo.toml [hunk 1]
```diff
@@ -56,6 +56,8 @@ fork = ["libafl_bolts/derive"]
 ## Collected stats to decide if observers must be serialized or not (which should reduce mem use and increase speed)
 adaptive_serialization = []
 
+## If this feature is set, `LibAFL` targets (and the fuzzer) will crash on `SIGPIPE` on unix systems.
+handle_sigpipe = []
```
Comment (tokatoka) [inline libafl/Cargo.toml:60]: should be default?

Comment (domenukk) [reply libafl/Cargo.toml:60]: Why?

Comment (domenukk) [reply libafl/Cargo.toml:60]: I don't think in normal fuzzing you ever want to catch sigpipe

Comment (tokatoka) [reply libafl/Cargo.toml:60]: nvm i read it wrong

**Comment by domenukk** [comment]
If anybody finds any issues with this let me know

**Pull request merged**